packages/connect-thegraph-token-manager/subgraph/src/aragon/aragon.ts

```diff
@@ -80,6 +80,7 @@ function _getAragonInfo(): AragonInfoEntity {
   if (!aragon) {
     aragon = new AragonInfoEntity(aragonId)
 
+    aragon.version = 'tokens-v0.1.0'
```

Comment:
This might be interesting for you @0xGabi 

For now, I'm not doing anything with this. It's just in case I ever doubt the version that is currently synced. However, we could have connectors look for this info and complain if they find a version that they don't know how to handle, or is deprecated.